src/template.rs

```diff
@@ -416,7 +481,8 @@ mod tests {
         let template = Template::try_from("nested {{ format }} %T").unwrap();
 
         assert_eq!(
-            Ok(Bytes::from("nested 2001-02-03 04:05:06")),
+            // Old behavior: Ok(Bytes::from("nested 2001-02-03 04:05:06")),
+            Ok(Bytes::from("nested %F 04:05:06")),
```

Comment:
Note, as discussed, that this technically is a breaking change here, though we have never advertised this behavior.